backend/src/main/java/com/example/backend/OrderingSysService.java

```diff
@@ -0,0 +1,76 @@
+package com.example.backend;
+
+import com.example.backend.dto.DishDTO;
+import com.example.backend.dto.DishInCartDTO;
+import com.example.backend.entity.Dish;
+import com.example.backend.entity.DishInCart;
+import com.example.backend.entity.Order;
+import com.example.backend.repo.DishInCartRepo;
+import com.example.backend.repo.DishRepo;
+import com.example.backend.repo.OrderRepo;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class OrderingSysService {
+    private final DishRepo dishRepo;
+    private final DishInCartRepo dishInCartRepo;
+    private final OrderRepo orderRepo;
```

Comment:
3 REpos? Das solltest du aufteilen in 3 verschiedene repos mit einem jeweiligen service